shell/assets/translations/en-us.yaml

```diff
@@ -7312,6 +7312,22 @@ branding:
     uploadLight: Upload Light Logo
     uploadDark: Upload Dark Logo
     useCustom: Use a Custom Logo
+  banner:
+    label: Banner
+    tip: 'Upload an image to replace the Rancher banner in the Dashboard page. Max file size is 200KB'
+    lightPreview: Light Theme Preview
+    darkPreview: Dark Theme Preview
+    uploadLight: Upload Light Banner
+    uploadDark: Upload Dark Banner
+    useCustom: Use a Custom Banner
+  loginBackground:
+    label: Login Background
+    tip: 'Upload an image to replace the Rancher background in the login page. Max file size is 200KB'
```

Comment:
Do we want to give any hints to dimensions that might work best for the banner and login background? I see that we already do this for the logo.

Reply:
Good catch, thanks. @kwwii do you a tip on how to calculate the recommended dimensions for banners and login background ?